projects/plugins/jetpack/extensions/blocks/business-hours/business-hours.php

```diff
@@ -105,11 +114,15 @@ function render( $attributes ) {
 		$attributes['days'] = get_default_days();
 	}
 
+	$wrapper_attributes = \WP_Block_Supports::get_instance()->apply_block_supports();
+
 	$start_of_week = (int) get_option( 'start_of_week', 0 );
 	$time_format   = get_option( 'time_format' );
 	$content       = sprintf(
-		'<dl class="jetpack-business-hours %s">',
-		! empty( $attributes['className'] ) ? esc_attr( $attributes['className'] ) : ''
+		'<dl class="jetpack-business-hours%s%s"%s>',
+		! empty( $attributes['className'] ) ? ' ' . esc_attr( $attributes['className'] ) : '',
```

Comment:
The `isset()` or `empty()` method is used because unlike JS, when accessing a non-existent property in PHP, a PHP warning will be triggered.

```
➜  jetpack git:(add/business-hours-block-design-tools) ✗ php -a
Interactive shell

php > $test = [];
php > echo $test['className'];
PHP Warning:  Undefined array key "className" in php shell code on line 1

Warning: Undefined array key "className" in php shell code on line 1

php > echo $test['className'] ? 'a' : 'b';
PHP Warning:  Undefined array key "className" in php shell code on line 1

Warning: Undefined array key "className" in php shell code on line 1
b
```

Reply:
I see. Makes sense now. Thank you for sharing the details Yan!